### ISSUE TITLE
The last test case is ignored in the report

### ISSUE THREAD

**Issue by ViktorMikhalkin** [Evilweed/protractor-beautiful-reporter#32]
After test run the result (PASS) of the last `it` within `describe` is absent in the report. In case when there is only one `it` then no report will be generated.

The environment is:
- java version "1.8.0_151"
- node version 8.8.1 (also was checked under 9.3.0)
- protractor version 5.2.2 (also was checked against 5.1.2 and 5.2.0)
- protractor-beautiful-reporter version is 0.5.6 (also was checked against 0.5.1 and 0.5.3)

The `package.json` is:
```
{
  "name": "prsample",
  "version": "0.0.1",
  "dependencies": {
    "protractor": "^5.2.2",
    "protractor-beautiful-reporter": "^0.5.6"
  }
}
```

The `conf.js` is:
```
exports.config = {
    framework: 'jasmine',
    seleniumAddress: 'http://localhost:4444/wd/hub',
    specs: ['spec.js'],
    onPrepare: function() {
        const HtmlReporter = require('protractor-beautiful-reporter');
        // Add a screenshot reporter and store screenshots to `/tmp/screenshots`:
        jasmine.getEnv().addReporter(new HtmlReporter({
            baseDirectory: 'tmp/screenshots'
        }).getJasmine2Reporter());
    }

};
```

The `spec.js` is:
```
describe('Protractor Demo App', function() {
    it('should have a title - 1', function() {
        browser.get('http://juliemr.github.io/protractor-demo/');

        expect(browser.getTitle()).toEqual('Super Calculator');
    });

    it('should have a title - 2', function() {
        browser.get('http://juliemr.github.io/protractor-demo/');

        expect(browser.getTitle()).toEqual('Super Calculator');
    });
});

**Comment by Evilweed** [comment]
@ViktorMikhalkin Fixed in [v0.5.7](https://github.com/Evilweed/protractor-beautiful-reporter/releases/tag/v0.5.7)

**Comment by ViktorMikhalkin** [comment]
@Evilweed Thank you! Now this works. Very, very fast. Appreciate this. Thank you once more!
